python.d/web_log.chart.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+# Description: web log netdata python.d module
+# Author: l2isbad
+
+from base import LogService
+import re
+import bisect
+from os import access, R_OK
+from os.path import getsize
+from collections import defaultdict, namedtuple
+
+priority = 60000
+retries = 60
+
+ORDER = ['response_codes', 'request_time', 'requests_per_url', 'http_method', 'bandwidth', 'clients_cur', 'clients_all']
+CHARTS = {
+    'response_codes': {
+        'options': [None, 'Response Codes', 'requests/s', 'responses', 'web_log.response', 'stacked'],
+        'lines': [
+            ['2xx', '2xx', 'absolute'],
+            ['5xx', '5xx', 'absolute'],
+            ['3xx', '3xx', 'absolute'],
+            ['4xx', '4xx', 'absolute'],
+            ['1xx', '1xx', 'absolute'],
+            ['0xx', 'other', 'absolute']
+        ]},
+    'bandwidth': {
+        'options': [None, 'Bandwidth', 'KB/s', 'bandwidth', 'web_log.bw', 'area'],
+        'lines': [
+            ['resp_length', 'received', 'absolute', 1, 1024],
+            ['bytes_sent', 'sent', 'absolute', -1, 1024]
+        ]},
+    'request_time': {
+        'options': [None, 'Processing Time', 'milliseconds', 'timings', 'web_log.request', 'area'],
+        'lines': [
+            ['resp_time_min', 'min', 'absolute', 1, 1],
+            ['resp_time_max', 'max', 'absolute', 1, 1],
+            ['resp_time_avg', 'avg', 'absolute', 1, 1]
+        ]},
+    'clients_cur': {
+        'options': [None, 'Current Poll Unique Client IPs', 'unique ips', 'unique clients', 'web_log.clients', 'line'],
```

Comment:
and this chart could be named `clients` and the context `web_log.clients` (as it is).